forwards_paper.tex

```diff
@@ -1167,43 +1167,56 @@ \section{Simplification algorithm}
 
 \end{taocpalg}
 
-The algorithm begins in \algref{S1} by allocating an empty list to store ancestral
+The algorithm begins in \algref{S0} by allocating an empty list to store ancestral
```

Comment:
Crap, I thought we were counting from 1 here. Alg W counts steps from 1. We need to fix one or the other I think. I don't mind whether we start with S0/W0 or S1/W1.

Reply:
/S[0-9]<cr><C>-An.n.n.n.